torch/csrc/api/include/torch/nn/module.h

```diff
@@ -383,15 +383,15 @@ class TORCH_API Module : public std::enable_shared_from_this<Module> {
 
   /// Serializes the `Module` into the given `OutputArchive`.
   ///
-  /// If the `Module` contains unserializable submodules (e.g. `nn::Functional`),
-  /// those submodules are skipped when serializing.
+  /// If the `Module` contains unserializable submodules (e.g.
+  /// `nn::Functional`), those submodules are skipped when serializing.
```

Comment:
nit: In general, I think we should avoid changing the formatting of unrelated comments in a PR, because when people try to look back at the history to see why a certain comment is added, they will land at this PR which doesn't actually tell them the reason (and they have to keep going back in history, which can add overhead).

Reply:
I'm sorry I have added PyTorch's ```clang-format``` to ```Qt Creator``` and it automatically formats the code on save. This is all PyTorch's ```clang-format``` which we're not enforcing at this moment.